### PR TITLE
Initial User Guide 

### PR DESCRIPTION
**Pull request by aedanmc**
Added our initial user guide to the repo. Doing this involved the following changes:

1. Adding USERDOC.md, which is the one-stop shop for all information related to using the project.
2. Moving the Synopsis section of the top-level README to USERDOC.md to reduce the amount of information that readers of the top-level README need to sift through initially.

### CODE REVIEW

File: documentation/instructions/USERDOC.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+# Installation and Running Travelo-Hey!
+
+As Travelo-Hey! is a web app, all that is required is JavaScript on either Chrome or Firefox. We are hosted on the Google Cloud Platform, so all you need to get started is to simply navigate to this link (**LINK TBD**)! If you want to run our development build, see our [build instructions](https://github.com/aedanmc/travelo-hey#Getting-Started).
+
+# Using Travelo-Hey!
+
+## Contents
+- [How do I get started?](#how-do-i-get-started)
```
Comment (C-Christensen) [inline documentation/instructions/USERDOC.md:30]: Currently this link is broken. Changing `#how-do-i-get-started` for `#Getting-started` would fix the issue.

File: README.md [hunk 2]
```diff
@@ -17,24 +17,13 @@
     - [Overview](#Overview)
     - [Responsibilities of Contributor](#Responsibilities-of-Contributor)
     - [Responsibilities of Maintainer](#Responsibilities-of-Maintainer)
-    - [Sanity Checks after Building Release](#Sanity-Checks-after-Building-Release)
+    - [Safety Checks after Building Release](#Sanity-Checks-after-Building-Release)
```
Comment (C-Christensen) [inline README.md:20]: I am sorry for not doing this updated in the developer doc PR. Can you please change the path from `#Sanity-Checks-after-Building-Release` to `#Safety-Checks-after-Building-Release` so the link works? Thank you.

**Review by C-Christensen — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aedanmc) [reply README.md]: I think I addressed this issue (and the other one you mentioned) with my latest commit. Can you check it again just to make sure, please?

Comment (C-Christensen) [reply README.md]: Thank you. It looks great!

**Review by C-Christensen — APPROVED** [review]
LGTM

**Review by mzuniga-1833381 — APPROVED** [review]
LGTM!

File: documentation/instructions/USERDOC.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# Using Travelo-Hey!
+
+## Contents
+- [How do I get started?](#getting-started)
+- [How do I search for a location?](#searching-for-a-location)
+- [How do I post a review?](#posting-a-review)
+- [How do I view a location's equality rating?](#viewing-a-locations-equality-rating)
+- [How do I see a country's overall safety rating?](#viewing-a-countrys-overall-safety-rating)
+
+### Getting started
```
Comment (mik3ycodez) [inline documentation/instructions/USERDOC.md:36]:
```suggestion
### Getting Started
```

File: documentation/instructions/USERDOC.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+To start looking at locations and gathering information for your next trip,
+you have a few options. From the landing page, you can either start searching for a location. You can also look at some of the places the Travelo-Hey! team features from the start by clicking on any of the location cards visible.
+
+### Searching for a location
```
Comment (mik3ycodez) [inline documentation/instructions/USERDOC.md:41]:
```suggestion
### Searching for a Location
```

File: documentation/instructions/USERDOC.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Searching for a location starts on the landing page using Travelo-Hey!'s filter options for country, city, and activity. Simply select at least one option from the three dropdown categories and click search. This will take you to a search results page containing locations that fit your filtered criteria. Clicking on a search result location card will take you to detailed information page and posted reviews about the location. 
+
+### Posting a review
```
Comment (mik3ycodez) [inline documentation/instructions/USERDOC.md:45]:
```suggestion
### Posting a Review
```

File: documentation/instructions/USERDOC.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+To post a review, start by accessing a location's specific page and scroll down below the information to the reviews section. To the left of the equality rating is a button labeled "Write a review". Clicking on this button will take you to a form fillout page where you can leave your review and add to the equality rating. Once filled out, click the "Submit" button on the bottom and your review will be posted on that location's page. 
+
+
+### Viewing a location's equality rating
```
Comment (mik3ycodez) [inline documentation/instructions/USERDOC.md:50]:
```suggestion
### Viewing a Location's Equality Rating
```

File: documentation/instructions/USERDOC.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Either by searching (work in progress) or just from one of Travelo-Hey!'s featured locations, click on a card displaying a business's profile image, name, phone number, and address. From there, our equality rating is displayed on the screen along with some reviews taken from Google's database of users.
+
+### Viewing a country's overall safety rating
```
Comment (mik3ycodez) [inline documentation/instructions/USERDOC.md:54]:
```suggestion
### Viewing a Country's Overall Safety Rating
```

**Review by mik3ycodez — APPROVED** [review]
LGTM

**Review by matthewdbroom13 — APPROVED** [review]

**Pull request merged**